.docker.env

```diff
@@ -1,6 +1,6 @@
 PHX_STATIC_PATH=/opt/app/rel/logflare/bin/priv/static
 DB_DATABASE=logflare_docker
-DB_HOSTNAME=localhost
+DB_HOSTNAME=db
```

Comment:
the db service in compose is exposed on host's `5432`, so both are fine.

Reply:
true but this way we are more explicit on what we're using